src/icon/petIcon.tsx

```diff
@@ -4,14 +4,14 @@ import isEmpty from 'lodash.isempty'
 
 import { color } from '_utils/branding'
 
-export enum PetType {
+enum Status {
   ON = 'on',
   OFF = 'off',
   DEFAULT = 'default',
```

Comment:
Why do we need the DEFAULT state?
IMO, the default state should be either ON or OFF and defined at the component call site, not a third state inside the component.
wdyt?

Reply:
We have 3 states for preferences (actually 4 since it can be "never changed" / "unselected" but in this case we don't display anything):
<img width="317" alt="Screen Shot 2019-04-04 at 14 11 40" src="https://user-images.githubusercontent.com/373381/55554744-d43f6480-56e3-11e9-9139-62ded5591b09.png">

Reply:
I see, thanks for the context.